function-support.html

```diff
@@ -5,12 +5,12 @@
     <meta name="viewport" content="initial-scale=1">
     <title>Function Support in KaTeX</title>
     <link rel="icon" href="http://khan.github.io/favicon.ico">
+	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.css" integrity="sha384-TEMocfGvRuD1rIAacqrknm5BQZ7W7uWitoih+jMNFXQIbNl16bO8OZmylH/Vi/Ei" crossorigin="anonymous">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/katex.min.js" integrity="sha384-jmxIlussZWB7qCuB+PgKG1uLjjxbVVIayPJwi6cG6Zb4YKq0JIw+OMnkkEC7kYCq" crossorigin="anonymous"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0/contrib/auto-render.min.js" integrity="sha384-IiI65aU9ZYub2MY9zhtKd1H2ps7xxf+eb2YFG9lX6uRqpXCvBTOidPRCXCrQ++Uc" crossorigin="anonymous"></script>
 
-	<link rel="stylesheet" href="https://unpkg.com/katex@0.9.0-beta/dist/katex.min.css">
-	<script src="https://unpkg.com/katex@0.9.0-beta/dist/katex.min.js"></script>
-	<script src="https://unpkg.com/katex@0.9.0-beta/dist/contrib/auto-render.min.js"></script>
-
-<style type="text/css">
+<style>

```

Comment:
Is there a reason that you dropped `type="text/css"` here?  I'm not sure it matters.

Reply:
Ah, interesting.  [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style) confirms that this is the default value, so definitely unnecessary.  Cool.